pkg/kubelet/volumemanager/metrics/metrics.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+func init() {
+	prometheus.MustRegister(volumeManagerVolumesMetric)
+	prometheus.MustRegister(reconstructVolumeErrorsMetric)
+}
+
+var (
+	volumeManagerVolumesMetric = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "volume_manager_total_volumes",
```

Comment:
I am not sure if this metric will be super useful without plugin name as a dimension.